Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_HeightAdjustTeleport.cs

```diff
@@ -15,25 +15,59 @@ namespace VRTK
     public class VRTK_HeightAdjustTeleport : VRTK_BasicTeleport
     {
         public bool playSpaceFalling = true;
+        public bool useGravity = true;
+        public float gravityFallHeight = 1.0f;
 
         private float currentRayDownY = 0f;
         private GameObject currentFloor = null;
+        private bool originalPlaySpaceFalling;
+        private bool isClimbing = false;
+
+        private VRTK_PlayerPhysics playerPhysics;
 
         protected override void Start()
         {
             base.Start();
             adjustYForTerrain = true;
+
+            originalPlaySpaceFalling = playSpaceFalling;
+
+            // Listen for climb events 
+            VRTK_PlayerClimb climbComponent = GetComponent<VRTK_PlayerClimb>();
```

Comment:
probably just var this

`var climbComponent = GetComponent<VRTK_PlayerClimb>();`

as you know the type is going to be of VRTK_PlayerClimb as it's obvious from the GetComponent.